docker/r/Dockerfile

```diff
@@ -88,6 +104,8 @@ RUN pip3 install --no-cache-dir pipx && \
     pipx inject renku sentry-sdk && \
     pipx ensurepath
 
+ENV PATH=/opt/conda/bin/conda:$PATH
```

Comment:
I think here you just want `ENV PATH=/opt/conda/bin:$PATH`

Reply:
might be a red herring:

In a Dockerfile in a project that looks like this:
```
FROM emmjab/renku-tests:latest
#FROM renku/singleuser-r:0.3.5-renku0.5.2

# Uncomment and adapt if code is to be included in the image
# COPY src /code/src

# install the python dependencies
COPY requirements.txt environment.yml /tmp/
RUN conda env update -q -f /tmp/environment.yml && \
    /opt/conda/bin/pip install -r /tmp/requirements.txt && \
    conda clean -y --all && \
    conda env export -n "root"
```
The CI/CD fails with:
```Step 1/3 : FROM emmjab/renku-tests:latest
 ---> 98772bb3e57f
Step 2/3 : COPY requirements.txt environment.yml /tmp/
 ---> Using cache
 ---> d5475087709c
Step 3/3 : RUN conda env update -q -f /tmp/environment.yml &&     /opt/conda/bin/pip install -r /tmp/requirements.txt &&     conda clean -y --all &&     conda env export -n "root"
 ---> Running in 39e75a0a7c6a
/bin/sh: 1: conda: not found
The command '/bin/sh -c conda env update -q -f /tmp/environment.yml &&     /opt/conda/bin/pip install -r /tmp/requirements.txt &&     conda clean -y --all &&     conda env export -n "root"' returned a non-zero code: 127
ERROR: Job failed: exit code 127```
even though the conda command is available when starting up a bash shell from the Docker image.

Reply:
Yes, it's available through bash in a login shell because the path is set via `. bashrc`. But in the case of a non-login shell you get the path set in the `ENV` which in this case points to the conda executable instead of the directory.

Reply:
I should have mentioned that I made the change to the path (add `/opt/conda/bin` to path instead of `/opt/conda/bin/conda`) before my last comment -- the Docker build with the user's conda file still fails. But at least that explains why it works anyway in `.bashrc`!